_includes/cockroachcloud/ccloud/backup-types.md

```diff
@@ -0,0 +1,4 @@
+{{ site.data.products.db }} clusters support two types of backups:
+
+- Managed-service backups: Cockroach Labs takes automated backups of {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} clusters that are stored in Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page for the type and frequency of backups for [{{ site.data.products.serverless }}](../cockroachcloud/backups-page.html?#backups-page) clusters and [{{ site.data.products.dedicated }}](../cockroachcloud/backups-page.html?filters=dedicated#backups-page) clusters
```

Comment:
Yes! Like this change! Couple small tweaks:

- Bold the names of each backup type
- Adjust the final sentence to remove the repeat of dedicated/serverless, so: 

> See the Restore Data from a Backup page for the type and frequency of backups for both {{ site.data.products.db }} clusters.